curriculum/challenges/english/25-front-end-development/lab-slice-and-splice/579e2a2c335b9d72dd32e05c.md

```diff
@@ -0,0 +1,91 @@
+---
+id: 579e2a2c335b9d72dd32e05c
+title: Implement the Slice and Splice Algorithm
+challengeType: 26
+dashedName: implement-the-slice-and-splice-algorithm
+---
+
+# --description--
+
+Create a `frankenSplice` function that accepts two arrays and an index.
+
+Copy each element of the first array into the second array, in order.
+
+Begin inserting elements at index `n` of the second array.
+
+Return the resulting array. The input arrays should remain the same after the function runs.
```

Comment:
```suggestion
In this lab, you will need to create an algorithm to merge two arrays.

Fulfill the user stories below and get all the tests to pass to complete the lab.

**User Stories:**

1. Create a `frankenSplice` function that accepts two arrays and an index.
1. Copy each element of the first array into the second array, in order, beginning at the given index, and return the resulting array.
1. The input arrays should remain the same after the function runs.
```

Following a similar pattern to what Jessica did here: https://github.com/freeCodeCamp/freeCodeCamp/pull/60409/files#diff-f31dc119b883391e4fe505a384391bbc6fef2a3d686e3c32028cff073460638bR10